openid-4-verifiable-presentations-1_0.md

```diff
@@ -1874,6 +1874,21 @@ Ecosystems that plan to leverage the trusted authorities mechanisms SHOULD make
 
 {backmatter}
 
+<reference anchor="JSON-LD" target="https://www.w3.org/TR/json-ld11/">
+  <front>
+    <title>Verifiable Credentials Data Model 1.1</title>
```

Comment:
Oh, Danny, this isn't VCDM. Is this VCDM? This isn't VCDM, is it? I didn't think so.

```suggestion
    <title>JSON-LD 1.1</title>
```